APMrover2/Rover.cpp

```diff
@@ -338,6 +338,54 @@ void Rover::publish_osd_info()
 }
 #endif
 
+/*
+  get the current wp index
+  return false if failed or n/a
+ */
+bool Rover::get_wp_index(uint16_t &index)
+{
+    if (control_mode == &mode_auto) {
```

Comment:
It's a little odd that Rover cares about the mode but Copter does not.

Reply:
this was to be consistent with how Rover::publish_osd_info() worked vs Copter::publish_osd_info()